proposals/0003-remove-var-parameters.md

```diff
@@ -4,7 +4,7 @@
 * Author: [David Farler](https://github.com/bitjammer)
 * Review Manager: [Joe Pamer](https://github.com/jopamer)
 * Status: **Implemented (Swift 3)**
-* Decision Notes: [Rationale](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160125/008145.html)
+* Decision Notes: [Rationale](https://forums.swift.org/t/se-0003-removing-var-from-function-parameters-and-pattern-matching/1230)
```

Comment:
The [latest archive](http://web.archive.org/web/20160414032941/https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160125/date.html) of the mailing list seems to have been saved on the Wayback Machine. Do you think it might be a good idea to add the Wayback Machine addresses in addition to the forums addresses? For example,

```
* Decision Notes: Rationale on [Swift Forums](https://forums.swift.org/t/se-0003-removing-var-from-function-parameters-and-pattern-matching/1230); [mailing list (archived)](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160125/008145.html)
```

* Decision Notes: Rationale on [Swift Forums](https://forums.swift.org/t/se-0003-removing-var-from-function-parameters-and-pattern-matching/1230); [mailing list (archived)](https://lists.swift.org/pipermail/swift-evolution/Week-of-Mon-20160125/008145.html)

Quoting from your post on the forums:

> it’s not that uncommon for a mailing list post to reference an earlier post by link.

I think adding Wayback Machine's addresses can help with it, because the links are redirected to their archived pages on Wayback Machine.

Reply:
> Do you think it might be a good idea to add the Wayback Machine addresses in addition to the forums addresses?

I don't know. I personally don't particularly like it, and I still don't understand why Apple can't keep these archives up indefinitely. It's good that the Wayback Machine has an archive (though it's hard to tell if it's complete) and you definitely have a point that its link rewriting would "preserve" internal links, but I only think of the Wayback Machine as a last resort if the original source has been lost. I'm still holding out hope that Apple can avoid this.